src/chart/pie/labelLayout.ts

```diff
@@ -473,7 +474,9 @@ export default function pieLabelLayout(
                 && labelPosition !== 'outer'
             ) {
                 labelRotate = radialAngle + Math.PI / 2;
-                if (labelRotate > Math.PI / 2) {
+                if (labelRotate > Math.PI / 2 && labelRotate < Math.PI * 1.5
```

Comment:
Clould you explain the check here? Also `Math.PI` can be saved to a variable here.